addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -29,6 +31,34 @@ def _get_proxy_urls(self):
         }
         return urls
 
+    @api.model
+    def _get_peppol_error_message(self, error_vals):
+        """
+        Helper to process the error dictionary returned from the IAP response.
+        It will only get the code (or EBMS code) and map it to the correct translated message.
+        :param dict error_vals: the dictionary of encoded error json generated from the `_json` method in `peppol_proxy`
+        :return: the translated error message
+        :rtype: str
+        """
+        default_error_message = _('Unknown Peppol Error: %s', error_vals)
+
+        if (ebms_code := error_vals.get('ebms_code')) and ebms_code != 4:
+            # Error with ebMS code is originally from PeppolInboundError
+            # In most case, ebMS message will be better and more specific, except for when the code is 4 (general "Other" message)
+            error_message = get_ebms_message(ebms_code, default_error_message)
```

Comment:
```suggestion
            error_message = get_ebms_message(ebms_code, default_message=default_error_message)
```
You can't be a fan of typing and forget to name your optional parameters :smirk: 
Joke aside I've seen very hard to debug mistakes because of that (signature changed over time and some calls made without optional parameter were just call with wrong parameters).